src/components/CoinsTable.jsx

```diff
@@ -32,14 +32,14 @@ const CoinsTable = () => {
   const navigate = useNavigate();
   const classes = useStyles();
 
-  const getcoins = async () => {
+  const getCoins = async () => {
     setLoading(true);
     const { data } = await axios.get(CoinList(currency));
     setCoins(data);
     setLoading(false);
   };
   useEffect(() => {
-    getcoins();
+    getCoins();
   }, [currency]);
   console.log(coins);
```

Comment:
Be sure to clean up any console.logs before releasing to your `main` branch. It's okay to keep them around for now if needed though for troubleshooting/functionality testing purposes.

Reply:
I will clean those console.logs later ,i just wanted to cheek that I am getting the right data!!